model/src/main/java/uk/gov/hmcts/payment/api/model/PaymentFeeLinkRepository.java

```diff
@@ -28,4 +29,11 @@ Optional<PaymentFeeLink> findByPaymentReferenceAndFeeId(final @Param("paymentRef
 
     @Query(value = "SELECT concat(date_part('year', now()),'-',nextval('payment_reference_seq'))", nativeQuery = true)
     String getNextPaymentReference();
+
+    @Query(value = "select ccd_case_number, count(ccd_case_number)\n" +
```

Comment:
is this right using \n in the "", haven't seen this before

Reply:
have removed